src/main/java/web/springboot/springbootproject/config/SecurityConfig.java

```diff
@@ -25,8 +30,14 @@ public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception
     }
 
     protected void configure(HttpSecurity http) throws Exception {
-        http.authorizeRequests()
-                .antMatchers("/register").permitAll()
+        http.httpBasic().disable()
+                .csrf().disable()
```

Comment:
do you need that since you have added bearer ?

Reply:
i found it on rescources

Reply:
i am not sure what i should change?